modules/mono/glue/GodotSharp/GodotSharp/Core/Dictionary.cs

```diff
@@ -449,6 +449,22 @@ private KeyValuePair<Variant, Variant> GetKeyValuePair(int index)
                 Variant.CreateTakingOwnershipOfDisposableValue(value));
         }
 
+        /// <summary>
+        /// Returns the entry at the given <paramref name="index"/>.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// <paramref name="index"/> is less than 0 or greater than the dictionary's size.
+        /// </exception>
+        /// <param name="index">The index of the entry to retrieve.</param>
+        /// <returns>The entry for the given <paramref name="index"/>.</returns>
+        public KeyValuePair<Variant, Variant> GetAt(int index)
+        {
+            if (index < 0 || index >= Count)
+                throw new ArgumentOutOfRangeException(nameof(index));
```

Comment:
With the added negative index support in the core/GDScript this is kinda inconsistent. @raulsntos Should this be changed as well or left as is?

Reply:
It should be left as is. The C# collections are more consistent with the .NET BCL than with GDScript. In C# you would use `System.Index` to access the elements from the end (e.g.: `array[^1]`).